st3/mdpopups/st_code_highlight.py

```diff
@@ -237,6 +237,13 @@ def set_view(self, src, lang, plugin_map):
             user_v = user_map.get(key, (tuple(), tuple()))
             if lang in (tuple(user_v[0]) + plugin_v[0] + v[0]):
                 for l in (tuple(user_v[1]) + plugin_v[1] + v[1]):
+                    if l.startswith('scope:'):
```

Comment:
So, I take this part is meant to allow users to describe their mapping in terms of scope?

Reply:
yes.

Reply:
Okay, the original description didn't highlight this. I had thought this was just a fallback. So the user is doing this?

````
```scope:source.js
content...
```
````

Reply:
Or is this used as a way in the mapping to link syntax. Forgive me, I haven't touched this area of the code a bit, and I haven't looked at the full context.

Reply:
> Okay, the original description didn't highlight this. I had thought this was just a fallback. So the user is doing this?
> 
> ```
> ```scope:source.js
> content...
> ```
> ```

Technically users can do this as it's a side effect of this PR. But that's not my purpose.

Reply:
Ah, okay, that makes sense. I'm not sure colons would make it through fenced code Markdown syntax anyways, though I honestly don't recall.

Reply:
> Okay, the original description didn't highlight this. I had thought this was just a fallback. So the user is doing this?
> 
> ```
> ```scope:source.js
> content...
> ```
> ```

That sets `lang` to `scope:source.js` and the fallback will match nothing as there is no scope `source.scope:source.js` or `text.scope:source.js`.